js/packages/cli/src/candy-machine-v2-cli.ts

```diff
@@ -87,6 +87,25 @@ programCommand('version', { requireWallet: false }).action(async () => {
   log.info(`Candy Machine Version: ${revision}`);
 });
 
+programCommand('versionIDL', { requireWallet: false }).action(
+  async (options, cmd) => {
+    const { env, rpcUrl } = cmd.opts();
+    //just to check one
+    const leakedKp = Keypair.fromSecretKey(
+      Uint8Array.from([
+        208, 175, 150, 242, 88, 34, 108, 88, 177, 16, 168, 75, 115, 181, 199,
+        242, 120, 4, 78, 75, 19, 227, 13, 215, 184, 108, 226, 53, 111, 149, 179,
+        84, 137, 121, 79, 1, 160, 223, 124, 241, 202, 203, 220, 237, 50, 242,
+        57, 158, 226, 207, 203, 188, 43, 28, 70, 110, 214, 234, 251, 15, 249,
+        157, 62, 80,
+      ]),
+    );
+
+    const anchorProgram = await loadCandyProgramV2(leakedKp, env, rpcUrl);
```

Comment:
```suggestion
    const anchorProgram = await loadCandyProgramV2(new Keypair(), env, rpcUrl);
```
I think generating a new keypair is a better option here.